### PR TITLE
Fixed issue 18 (Add custom method feature a bit confusing).

### PR DESCRIPTION
**Pull request by Ashley-Waldron**
Added CSS to highlight border on hover and updated the alt text.

### CODE REVIEW

File: partials/main.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h2 class="panel-title">Request Settings</h2>
 													<a href="" ng-click="changeRequestMethod(type)" ng-hide="requestMethod.selected === type">{{type}}</a>
 												</li>
 												<li>
-													<input type="text" placeholder="Custom" form="" ng-model="requestMethod.custom" ng-keyup="$event.keyCode == 13 && changeRequestMethod(requestMethod.custom)" title="Enter a custom request method">
+													<input type="text" placeholder="Custom" form="" ng-model="requestMethod.custom" ng-keyup="$event.keyCode == 13 && changeRequestMethod(requestMethod.custom)" title="Click here to enter a custom request method">
```
Comment (glencollins18888) [inline partials/main.html:28]: Good work !
Do you need localization for this ?:)

**Comment by paulhitz** [comment]
Nice one Ashley! :)

File: partials/main.html (hunk 1 above)
Comment (paulhitz) [inline partials/main.html:28]: No, we're just a bunch of cowboys :)

**Comment by Ashley-Waldron** [comment]
I'll leave the localization to Paul, he loves that sorta stuff!

**Pull request merged**